freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/create/FreeIpaRecommendationResponse.java

```diff
@@ -0,0 +1,43 @@
+package com.sequenceiq.freeipa.api.v1.freeipa.stack.model.create;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import io.swagger.annotations.ApiModel;
+
+@ApiModel("FreeIpaRecommendationV1Response")
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class FreeIpaRecommendationResponse {
```

Comment:
please implement `toString` method for this and all of the request/response objects